### PR TITLE
Update set-up-resource-account.md

### PR DESCRIPTION
**Pull request by AlexanderHolmeset**
org wide settigns menu does not excist anymore

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/business-voice/set-up-resource-account.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ After you've created your resource account, you need to assign a *Microsoft 365
 ![Screenshot of the assign service number user interface.](../media/resource-account-assign-phone-number.png)
 
 1. Open the Microsoft Teams admin center and log in with a user that is a Global admin (this is usually the account you used to sign up for Microsoft 365).
-1. In the left navigation pane, go to <a href="https://admin.teams.microsoft.com/company-wide-settings/resource-accounts" target="_blank">**Org-wide settings** > **Resource accounts**</a>.
+1. In the left navigation pane, go to <a href="https://admin.teams.microsoft.com/company-wide-settings/resource-accounts" target="_blank">**Voice** > **Resource accounts**</a>.
 1. Select the resource account you just created, and then click **Assign/unassign**.
 1. In the **Phone number type** dropdown, choose **Online**.
 1. In the **Assigned phone number** box, search for the number you want to use and click **Add**. Be sure to include the country code (for example, **+1** 250 555 0012)
```
Comment (JohanFreelancer9) [inline Teams/business-voice/set-up-resource-account.md:89]:
```suggestion
1. In the **Assigned phone number** box, search for the number you want to use and click **Add**. Be sure to include the country code (for example, **+1** 250 555 0012).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AlexanderHolmeset, kindly commit the minor additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@tiburd, could you merge it? It's a very simple steps update in the Teams Admin Center, I have verified it.

Do we need to wait for @dstrome approval?

**Comment by tiburd** [comment]
reviewed + confirmed + fixed file conflict

**Pull request merged**